### ISSUE TITLE
Installing Problem

### ISSUE THREAD

**Issue by vahidkaargar** [rinvex/laravel-auth#151]
Installing problem on latest version of laravel

 Problem 1
    - laravel-notification-channels/authy dev-develop requires illuminate/notifications ~5.6.0 -> satisfiable by laravel/framework[v5.6.16, 5.6.x-dev, v5.6.0, v5.6.1, v5.6.10, v5.6.11, v5.6.12, v5.6.13, v5.6.14, v5.6.15, v5.6.2, v5.6.3, v5.6.4, v5.6.5, v5.6.6, v5.6.7, v5.6.8, v5.6.9], illuminate/notifications[5.6.x-dev, v5.6.0, v5.6.1, v5.6.10, v5.6.11, v5.6.12, v5.6.13, v5.6.14, v5.6.15, v5.6.16, v5.6.2, v5.6.3, v5.6.4, v5.6.5, v5.6.6, v5.6.7, v5.6.8, v5.6.9].
    - don't install laravel/framework v5.6.4|don't install laravel/framework 5.5.x-dev
    - don't install laravel/framework v5.6.5|don't install laravel/framework 5.5.x-dev
    - don't install laravel/framework v5.6.6|don't install laravel/framework 5.5.x-dev
    - don't install laravel/framework v5.6.7|don't install laravel/framework 5.5.x-dev
    - Can only install one of: laravel/framework[5.6.x-dev, 5.5.x-dev].
    - Can only install one of: laravel/framework[v5.6.0, 5.5.x-dev].
    - Can only install one of: laravel/framework[v5.6.1, 5.5.x-dev].
    - Can only install one of: laravel/framework[v5.6.10, 5.5.x-dev].
    - Can only install one of: laravel/framework[v5.6.11, 5.5.x-dev].
    - Can only install one of: laravel/framework[v5.6.12, 5.5.x-dev].
    - Can only install one of: laravel/framework[v5.6.13, 5.5.x-dev].
    - Can only install one of: laravel/framework[v5.6.14, 5.5.x-dev].
    - Can only install one of: laravel/framework[v5.6.15, 5.5.x-dev].
    - Can only install one of: laravel/framework[v5.6.16, 5.5.x-dev].
    - Can only install one of: laravel/framework[v5.6.2, 5.5.x-dev].
    - Can only install one of: laravel/framework[v5.6.3, 5.5.x-dev].
    - Can only install one of: laravel/framework[v5.6.8, 5.5.x-dev].
    - Can only install one of: laravel/framework[v5.6.9, 5.5.x-dev].
    - Can only install one of: laravel/framework[5.5.x-dev, v5.6.16].
    - Can only install one of: laravel/framework[5.5.x-dev, v5.6.16].
    - don't install illuminate/notifications 5.6.x-dev|don't install laravel/framework 5.5.x-dev
    - don't install illuminate/notifications v5.6.0|don't install laravel/framework 5.5.x-dev
    - don't install illuminate/notifications v5.6.1|don't install laravel/framework 5.5.x-dev
    - don't install illuminate/notifications v5.6.10|don't install laravel/framework 5.5.x-dev
    - don't install illuminate/notifications v5.6.11|don't install laravel/framework 5.5.x-dev
    - don't install illuminate/notifications v5.6.12|don't install laravel/framework 5.5.x-dev
    - don't install illuminate/notifications v5.6.13|don't install laravel/framework 5.5.x-dev
    - don't install illuminate/notifications v5.6.14|don't install laravel/framework 5.5.x-dev
    - don't install illuminate/notifications v5.6.15|don't install laravel/framework 5.5.x-dev
    - don't install illuminate/notifications v5.6.16|don't install laravel/framework 5.5.x-dev
    - don't install illuminate/notifications v5.6.2|don't install laravel/framework 5.5.x-dev
    - don't install illuminate/notifications v5.6.3|don't install laravel/framework 5.5.x-dev
    - don't install illuminate/notifications v5.6.4|don't install laravel/framework 5.5.x-dev
    - don't install illuminate/notifications v5.6.5|don't install laravel/framework 5.5.x-dev
    - don't install illuminate/notifications v5.6.6|don't install laravel/framework 5.5.x-dev
    - don't install illuminate/notifications v5.6.7|don't install laravel/framework 5.5.x-dev
    - don't install illuminate/notifications v5.6.8|don't install laravel/framework 5.5.x-dev
    - don't install illuminate/notifications v5.6.9|don't install laravel/framework 5.5.x-dev
    - rinvex/fort v0.0.6 requires laravel-notification-channels/authy dev-develop -> satisfiable by laravel-notification-channels/authy[dev-develop].
    - rinvex/fort v0.0.6 requires laravel/framework ~5.5.0 -> satisfiable by laravel/framework[5.5.x-dev].
    - Installation request for rinvex/fort ^0.0.6 -> satisfiable by rinvex/fort[v0.0.6].

Installation failed, reverting ./composer.json to its original content.

**Comment by vahidkaargar** [comment]
thanks, its helped.

**Comment by vahidkaargar** [comment]
in installation steps i have this error after step 4: (php artisan migrate --path="vendor/rinvex/fort/database/migrations")


ERROR:

```
In ProviderRepository.php line 208:

  Class 'Rinvex\Fort\Providers\FortServiceProvider' not found
```

**Comment by noxify** [comment]
Hi,

The readme is a bit outdated.

Please use `php artisan rinvex:publish:auth`.

The command will publish the config file.

**Comment by noxify** [comment]
FYI

The package `rinvex/fort` has been renamed to `rinvex/auth` and there are many breaking changes (e.g. there are no migrations in the current develop branch).

If you need these migration files, please use the v0.0.6, but this version requires laravel 5.5.

**Comment by Omranic** [comment]
Closing since this is outdated, and should be fixed in latest version.
Feel free to open a new issue if you've further questions.
